cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DistributedStorageProviderNamespaceAdmin.java

```diff
@@ -69,10 +73,11 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
     // TODO: CDAP-1519: Create base directory for filesets under namespace home
     // create HBase namespace
     String hbaseNamespace = tableUtil.getHBaseNamespace(namespaceMeta);
-    try (HBaseAdmin admin = new HBaseAdmin(hConf)) {
+    try (HBaseDDLExecutor executor = hBaseDDLExecutorFactory.get();
```

Comment:
remove this try and put it just where we use it to call HBaseAdmin.hasNamespace() to avoid creating unnecessarily